java-checks-test-sources/src/main/java/checks/security/DisclosingTechnologyFingerprints.java

```diff
@@ -0,0 +1,91 @@
+package checks.security;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpServletResponseWrapper;
+import org.apache.wicket.protocol.http.BufferedWebResponse;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+public class DisclosingTechnologyFingerprints {
```

Comment:
We should add a test for `org.rapidoid.http.Resp` even if it has to be in a non-compiling source file